plugin/sampling/strategyprovider/adaptive/post_aggregator.go

```diff
@@ -398,7 +401,7 @@ func (p *PostAggregator) isUsingAdaptiveSampling(
 	// before.
 	if len(p.serviceCache) > 1 {
 		if e := p.serviceCache[1].Get(service, operation); e != nil {
-			return e.UsingAdaptive && !FloatEquals(e.Probability, p.InitialSamplingProbability)
+			return !FloatEquals(e.Probability, p.InitialSamplingProbability)
```

Comment:
@yurishkuro with this patch, the numbers seem to make a bit more sense. here's the output i see now 
```
"post_aggregator_service_cache[0]": "map[tracegen:map[lets-go:{1 true}]]"
"post_aggregator_service_cache[0]": "map[tracegen:map[lets-go:{0.20840949054166666 false}]]"
"post_aggregator_service_cache[0]": "map[tracegen:map[lets-go:{0.021478798306074933 false}]]"
"post_aggregator_service_cache[0]": "map[tracegen:map[lets-go:{0.012634530863339348 false}]]"
"post_aggregator_service_cache[0]": "map[tracegen:map[lets-go:{0.02105483853348014 false}]]"
….
"post_aggregator_service_cache[0]": "map[tracegen:map[lets-go:{0.08421935413392057 false}]]"
"post_aggregator_service_cache[0]": "map[tracegen:map[lets-go:{0.1403881259748575 false}]]"
"post_aggregator_service_cache[0]": "map[tracegen:map[lets-go:{0.08254900503950167 false}]]"
"post_aggregator_service_cache[0]": "map[tracegen:map[lets-go:{0.051602341629876265 false}]]"
```

Reply:
let me know if you have any thoughts on how to proceed here

Reply:
* `post_aggregator_service_cache` name is unclear, are these probabilities, or throughput?
* the Boolean value at the end looks suspicious, what does it mean? If it's "using adaptive sampling" indicator then we need to know why it goes to false.

Reply:
- its the sampling cache entry where the first value is the probability and the second is the using adaptive sampling indicator
- i mostly reverted the old `isAdaptiveSamplingLogic` aside from the e.UsingAdaptive check. When I had it hardcoded to true, the values didn't make sense (see https://github.com/jaegertracing/jaeger/pull/5951#issuecomment-2395246968)